### ISSUE TITLE
[Question] Error: Cannot find module './myReporter.ts' / MODULE_NOT_FOUND

### ISSUE THREAD

**Issue by alanbqs** [microsoft/playwright#12309]
I am trying to implement a custom reporter as mentioned on https://playwright.dev/docs/test-reporters#custom-reporters. I have created a file 'myReporter.ts' with the supplied typeScript, and have saved this in the same location as my tests.

I have modified my playwright.config.ts file to change the reporter to ```reporter: './myReporter.ts',``` 

But when I run my test I get 

```
C:\playwright\myproj>cd C:\playwright\myproj && npx playwright test test.spec.ts.datamerged.ts  --project=chromium
Using config at C:\playwright\myproj\playwright.config.ts
Error: Cannot find module './myReporter.ts'
Require stack:
- C:\playwright\myproj\node_modules\@playwright\test\lib\loader.js
- C:\playwright\myproj\node_modules\@playwright\test\lib\runner.js
- C:\playwright\myproj\node_modules\@playwright\test\lib\cli.js
- C:\playwright\myproj\node_modules\playwright-core\lib\cli\innerCli.js
- C:\playwright\myproj\node_modules\playwright-core\lib\cli\cli.js
- C:\playwright\myproj\node_modules\playwright-core\cli.js
- C:\playwright\myproj\node_modules\@playwright\test\cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:925:15)
    at Function.resolve (node:internal/modules/cjs/helpers:98:19)
    at C:\playwright\myproj\node_modules\@playwright\test\lib\loader.js:443:21
    at Array.map (<anonymous>)
    at resolveReporters (C:\playwright\myproj\node_modules\@playwright\test\lib\loader.js:441:110)
    at Loader._processConfigObject (C:\playwright\myproj\node_modules\@playwright\test\lib\loader.js:114:88)
    at Loader.loadConfigFile (C:\playwright\myproj\node_modules\@playwright\test\lib\loader.js:81:10)
    at loadConfig (C:\playwright\myproj\node_modules\@playwright\test\lib\runner.js:91:24)
    at loadConfigFromDirectory (C:\playwright\myproj\node_modules\@playwright\test\lib\runner.js:98:24)
    at Runner.loadConfigFromFile (C:\playwright\myproj\node_modules\@playwright\test\lib\runner.js:107:22)
    at runTests (C:\playwright\myproj\node_modules\@playwright\test\lib\cli.js:148:18)
    at Command.<anonymous> (C:\playwright\myproj\node_modules\@playwright\test\lib\cli.js:73:7) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\playwright\\myproj\\node_modules\\@playwright\\test\\lib\\loader.js',
    'C:\\playwright\\myproj\\node_modules\\@playwright\\test\\lib\\runner.js',
    'C:\\playwright\\myproj\\node_modules\\@playwright\\test\\lib\\cli.js',
    'C:\\playwright\\myproj\\node_modules\\playwright-core\\lib\\cli\\innerCli.js',
    'C:\\playwright\\myproj\\node_modules\\playwright-core\\lib\\cli\\cli.js',
    'C:\\playwright\\myproj\\node_modules\\playwright-core\\cli.js',
    'C:\\playwright\\myproj\\node_modules\\@playwright\\test\\cli.js'
  ]
}
npm ERR! code 1
npm ERR! path C:\playwright\myproj
npm ERR! command failed
```

What have I missed?

**Comment by ryanrosello-og** [comment]
have you tried compiling your  'myReporter.ts' to javascript? and then point your config to the js file?

**Comment by mxschmitt** [comment]
@alanbqs Which Playwright version are you using?

I remember a similar bug which should be fixed in 1.19

**Comment by alanbqs** [comment]
I'm on v1.18.1

**Comment by mxschmitt** [comment]
Try 1.19.1 please.

**Comment by mxschmitt** [comment]
This is expected, the reporter is resolved relative to the configuration file. The new error which you getting is because you don't have browsers installed, so you need to execute `npx playwright install`.

**Comment by alanbqs** [comment]
But I have already run the install but this missing file is still missing; although there is a ".../chromium-956323/chrome-win/chrome.exe")

**Comment by mxschmitt** [comment]
Maybe you have multiple Playwright installations installed?

Try to only put `@playwright/test: 1.19.2` inside your package.json, removing the node_modules folder and running npm install.

**Comment by alanbqs** [comment]
Ok - I've resolved the ```browserType.launch: Executable doesn't exist at ``` issue by changing the php service login from [Windows] ```Local System account``` to a specific login account (I don't know why Local System doesn't see the same as my login. (I had to go into the windows Services list and edit the php process properties to set the [Log On]).

I'm not familiar with npx/npm but I wonder if there's an issue with where I ran the command from - resulting in different installations across the PC.

I've also removed my playwright folder and re-installed playwright inside my php wamp/www/myProj folder. 

I've also upgraded to 1.19.2

With all that - I am now getting the myReporter.ts reporter being triggered - so hurrah... something has fixed it.
